resources/carrier/en/351.txt

```diff
@@ -13,14 +13,37 @@
 # limitations under the License.
 
 35191|Vodafone
-351922|TMN
-3519240|TMN
-3519241|TMN
-3519242|TMN
-3519243|TMN
-3519244|TMN
-351925|TMN
-351926|TMN
-351927|TMN
-35193|Optimus
-35196|TMN
+3519200|Lycamobile
+3519201|Lycamobile
+3519202|Lycamobile
+3519203|Lycamobile
+3519204|Lycamobile
+3519205|Lycamobile
+351921|Vodafone
+3519220|CTT
+3519221|CTT
+3519222|CTT
+3519222|CTT
```

Comment:
Please remove duplicate entries of '3519222|CTT'

Reply:
Done!